mne/channels/tests/test_interpolation.py

```diff
@@ -323,7 +323,7 @@ def test_interpolation_nirs():
     raw_od.interpolate_bads()
     assert raw_od.info["bads"] == []
     assert bad_0_std_pre_interp > np.std(raw_od._data[bad_0])
-    raw_haemo = beer_lambert_law(raw_od, ppf=6)
+    raw_haemo = beer_lambert_law(raw_od, ppf=(6.0, 6.0))
```

Comment:
None of these changes should be needed -- let's only change it in one place in one test maybe